### PR TITLE
style radio buttons on manage profile settings page =D

### PR DESCRIPTION
**Pull request by ahsu1230**
Thanks @yipingliao for the advice before!
@andrewconner @2is10 

and removing some things from twitter connect controller now that `profileUrl` is in

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/profile/profile.styl [hunk 1]
```diff
@@ -268,44 +268,59 @@ input.profile-last-name
...
-    border-radius: 50%
+    width: 8px
+    height: 8px
     background-color: darkBlueColor
+    border-radius: 50%
+    content: ' '
 
-.profile-manage-settings-option-text
-  cursor: pointer
+.profile-manage-input.profile-manage-input:checked ~ .profile-manage-radio-check:after
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/src/profile/profile.styl:316]: This selector can be shortened to `:checked + .profile-manage-radio-check::after`. `+` is for adjacent siblings. Also, a single colon works to create pseudoelements, but we prefer the standard two colons (`::before` and `::after`).

**Comment by 2is10** [comment]
:+1:

**Comment by yipingliao** [comment]
I may be remembering the wrong thing, but did this change help with those `_.isBoolean()` checks from before?

**Comment by ahsu1230** [comment]
unfortunately not. the watch still picks up the changed values from `undefined` to whatever initial value they should be in

**Pull request merged**